src/main/java/Bank.java

```diff
@@ -0,0 +1,60 @@
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.Map;
+
+public class Bank {
```

Comment:
Naming is not right, this is actual response from API call, you are interested only in rates, map them to other object like Currency, which can have something like: id, currencyCode, rate, updatedOn. Because we don't want to save such data in our local projection